### PR TITLE
[337] Limit 2 concurrent environments for free accounts

### LINKED ISSUE #337: Phase 3 Billing: Limit 2 concurrent environments for free accounts
Core part of https://github.com/UffizziCloud/uffizzi_platform/issues/588

### PR DESCRIPTION
**Pull request by antlu**
Closes #337

### CODE REVIEW

File: core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments/activity_items_controller.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ class UffizziCore::Api::Cli::V1::Projects::Deployments::ActivityItemsController
   # @response 401 Not authorized
   # @response 404 Not found
   def index
-    deployment = resource_project.deployments.existed.find(params[:deployment_id])
+    deployment = resource_project.deployments.actual.find(params[:deployment_id])
```
Comment (zipofar) [inline core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments/activity_items_controller.rb:18]: Why have we changed from `existed` to `actual`?

File: core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller.rb [hunk 2]
```diff
@@ -160,7 +160,7 @@ def check_credentials(compose_file)
   end
 
   def deployments
-    @deployments ||= resource_project.deployments.existed
+    @deployments ||= resource_project.deployments.actual
```
Comment (zipofar) [inline core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller.rb:163]: Why have we changed from existed to actual?

File: core/app/repositories/uffizzi_core/deployment_repo.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ module UffizziCore::DeploymentRepo
       where(name: name)
     }
     scope :with_amazon_repos, -> { includes(containers: [:repo]).where(containers: { repos: { type: UffizziCore::Repo::Amazon.name } }) }
-    scope :existed, -> { where(state: [:active, :failed]) }
+    scope :actual, -> { where(state: [:active, :failed]) }
```
Comment (zipofar) [inline core/app/repositories/uffizzi_core/deployment_repo.rb:11]: Why have we changed from `existed` to `actual`?

Comment (antlu) [reply core/app/repositories/uffizzi_core/deployment_repo.rb:11]: It is more descriptive than the past form of "exist".
Why not existing: `disabled` are existing too.

**Review by zipofar — APPROVED** [review]

**Review by moklidia — APPROVED** [review]

**Review by zipofar — APPROVED** [review]

**Pull request merged**